spy_terminal.py

```diff
@@ -27,76 +30,94 @@ def main():
         if decision == 'Y':
-            entry = input("Please give me your message. Note: Do not use numbers: ")
+            entry = input(
+                "Please give me your message. Note: Do not use numbers: ")
             scrambled_message = scramble(entry, codex)
             return scrambled_message
     if count < 1:
-        entry = input("Please give me your message. Note: Do not use numbers: ")
+        entry = input(
+            "Please give me your message. Note: Do not use numbers: ")
         scrambled_message = scramble(entry, codex)
         return scrambled_message
 
+
 '''This function takes in an existing password, a coded message and the codex. 
     It determines whether the password is correct and if so unscrambles the message and prints it for the user.
     A count lets user know if no messages have been left. 
     It also erases the message automatically'''
-def retrieve_message(password,coded_message,codex,count):
+
+
+def retrieve_message(password, coded_message, codex, count):
     if count == 0:
         print("Currently no saved messages.")
         return
     if check_identity(password):
-         security_code = random.randint(0,999999)
-         verification_code = 0
-         while verification_code != security_code:
+        security_code = random.randint(0, 999999)
+        verification_code = 0
+        while verification_code != security_code:
             verification_code = email_verify(security_code)
-         if verification_code == security_code:
-             fixed_message = unscramble(coded_message, codex)
-             print("Your message has been unscrambled!" + "\n" + "You message is: " + fixed_message +
-                   "\n" + "This message will self-erase in 10 seconds.")
-             self_destruct(10, "Message Erased", "no")
-             os.system(SYSTEM)
+        if verification_code == security_code:
+            fixed_message = unscramble(coded_message, codex)
+            print("Your message has been unscrambled!" + "\n" + "You message is: " + fixed_message +
+                  "\n" + "This message will self-erase in 10 seconds.")
+            self_destruct(10, "Message Erased", "no")
+            os.system(SYSTEM)
+
 
 '''This code sends an email to the user's email address which contains a random security password. It then asks the user
 to input this code before progressing.'''
+
+
 def email_verify(security_code):
     check = 'no'
     while check == 'no':
-        email_address = str(input("Enter your email address for ID confirmation: "))
+        email_address = str(
+            input("Enter your email address for ID confirmation: "))
         sender_email = "cipsecretagency@gmail.com"
         receiver_email = email_address
         subject = "Agent Identity Confirmation"
-        text = "Good Day: Your confirmation code is " + str(security_code) + "!"
+        text = "Good Day: Your confirmation code is " + \
+            str(security_code) + "!"
```

Comment:
Python 3.6+ released f-strings to make it easier to format strings with dynamic variables embedded
```suggestion
        text = f'Good Day: Your confirmation code is {security_code}!'
```